src/java/org/apache/cassandra/config/JMXServerOptions.java

```diff
@@ -52,7 +51,7 @@ public class JMXServerOptions
     public final Boolean authenticate;
 
     // ssl options
-    public final EncryptionOptions jmx_encryption_options;
+    public final EncryptionOptions.ServerEncryptionOptions jmx_encryption_options;
```

Comment:
@smiklosovic I feel this should be `ClientEncryptionOptions` because today this settings are used to specify the ssl configuration used by the JMX Server to communicate with the remote JMX ssl client.